resources/packs/aws/aws_cloudtrail.go

```diff
@@ -119,9 +121,14 @@ func (t *mqlAwsCloudtrail) getTrails(provider *aws_provider.Provider) []*jobpool
 				if regionVal != core.ToString(trail.HomeRegion) {
 					continue
 				}
-
+				// we skip trails that are not coming from the account that is being scanned
+				// such trails are organization trails, created from another (master) acc in the org
+				trailArn := core.ToString(trail.TrailARN)
+				if !strings.Contains(trailArn, acc) {
```

Comment:
is there any information we can gather about organization trails?  we should be able to work around not being able to fetch a kms key or log group, since keys can be deleted anytime, it's always a problem we can run into

Reply:
yeah, we can also just not err out if the key or the log group are not found.

re the org trails: there's a bool on the trail that indicates what it is. i am not sure if we can use it since i'd assume we want to show that trail in the master acc it has been created from